### DISCUSSION TITLE
Old asset versions restored / updated? (Source 3D models?) [Asset Colour Variations]

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#915] [Q&A]
https://opengameart.org/content/isometric-hero-and-creatures
I am not sure where to find the source files / .blend files for these character variants. The 'Plate Rogue' might look cool with Chain Mail textures. Also if the .blend files could be found perhaps they could be increase in resolution (4k even?) and be used as alternative character sprite or at least NPC / Mob sprites (since this is a thing now hopefully: https://github.com/flareteam/flare-engine/issues/1802 ) / other uses to hopefully add visual variety.

Anyone have the source / .blend files for these tilesets?
---------------
Light (2009):
https://www.youtube.com/watch?v=1rHEBC0mxAo&ab_channel=ClintBellanger
Grey (2010):
https://www.youtube.com/watch?v=cxTGtWV1H8Y&ab_channel=ClintBellanger
https://www.youtube.com/watch?v=slTp_dWuRFM&ab_channel=ClintBellanger
Shadow:
https://www.youtube.com/watch?v=3AhZrwJ8CVs&ab_channel=ClintBellanger
https://www.youtube.com/watch?v=qAJ7ogpcsj4&ab_channel=ClintBellanger
Dark:
https://www.youtube.com/watch?v=K-zQF8PnOMI&ab_channel=ClintBellanger
https://www.youtube.com/watch?v=bSJavaVHWpw&ab_channel=ClintBellanger
Sandstone:
https://www.youtube.com/watch?v=M_Rti-iiUVA&ab_channel=ClintBellanger
https://www.youtube.com/watch?v=c8Byu-IirLI&ab_channel=ClintBellanger
https://www.youtube.com/watch?v=dCCLhE4QDe8&ab_channel=ClintBellanger

Is there a way to find the .blend files for these and render them in 4K (and hopefully keep their theme / tone / colouration)?
Diablo 1 had four tilesets (6 including hellfire) yet each tileset had 4 to 5 colour variations to increase a sense of visual diversity / mix it up / add visual variety:
![Dungeon_Variations](https://user-images.githubusercontent.com/86871191/178848682-9e570b84-7d53-4dac-8ac1-f6ff1114d258.png)
![Catacombs_Variations](https://user-images.githubusercontent.com/86871191/178848722-27878ee0-4b68-4e26-8547-c855f0a530e8.png)
![Caves_Variations](https://user-images.githubusercontent.com/86871191/178848744-b12f675b-390b-4f7f-a9c7-5bd0d205c4a0.png)
![Hell_Variations](https://user-images.githubusercontent.com/86871191/178848765-5d7a3be5-c3f4-4d0f-9ad6-d979a145f31f.png)

On a personal level I would like to have a (Light -> Sand Stone -> Grey / Dark -> Shadow) visual level theme progression for a large scale catacombs dungeon with many levels; 33+1 ground level to be exact. I am wondering if anyone else has thought of colour variations and or to restore / update / have the source .blend files for the older art assets.

Also I vaguely remember that some older .blend files only work with older version of blender. If that is the case how can forward porting work with keeping the details (if that is even an issue in this instance)? Something something lacking ~'alpha transparency' or something super old Flare assets; I am not able to fully remember that point fully. I hope that there is a way to save / restore older styles of Flare so that there are more options to tinker with and have more assets brought up to high resolution / high quality standards.

I remember searching around the old Flare Github / Clint's OpenGameArt.org profile and finding like .png images of tilesets and sprites but coming up short on finding the older versions source models / .blend files. I am really hoping to save / find the older versions and hopefully find a way to bring them up to current visual quality standards / formats. I am really hoping to find the 'Shadow', 'Dark' and 'Grey' tilesets .blend files and hopefully bring them up to higher resolutions / even add / ~'back port' more assets to match the visual themes / colourations (e.g. chests and later added decorations).

I have found the 0.14 Flare Dungeon / Tilesets https://github.com/clintbellanger/flare/tree/branch0.14/art/tilesets/dungeon but I am not sure where the 0.6 and 0.10 (Shadow) [and earlier] are found / how to obtain them (wasn't there a pre-Github Flare / OSARE). Also the old flarerpg.org blog download links are broken does anyone have a back up / mirror / archive willing to share e.g. https://flarerpg.org/2010/01/01/20100101/ @clintbellanger ? Also the 0.14 tileset .blend files seem to work with Blender 2.79 yet I am not 100% sure how to get them to render in 4K off the top of my head. I figured out how to make tiles in 8 directions before I should be able to tinker with it but help is greatly appreciated.

Basically I am wondering how to get the old assets ~'remade' / ~'rerendered' into new assets with back ported assets included?
